workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs

```diff
@@ -7,19 +7,19 @@ namespace Improbable.Gdk.Core
 {
     public abstract class WorkerBase : IDisposable
     {
-        public World World { get; private set; }
+        public World World { get; }
```

Comment:
Is this semantically the same?

Reply:
It's not, but since nothing actually ever assigns to these properties outside of a constructor, it's best to leave it read-only until a setter is needed.

Reply:
Sgtm 👍